### DISCUSSION TITLE
React hook form controller and ref

### DISCUSSION THREAD

**Discussion by markotufe** [facebook/lexical#4376] [Q&A]
Hello,

React hook form(RHF) controller gives us ref to attach to input. RHF can focus and scroll to input when is invalid. 

Is it possible to add that ref to lexical rich text editor?
I created that behaviour with editor instance using editor.focus and custom ref using scrollRef.current.scrollIntoView. This is separate from RHF. I want to RHF controll that behaviour, is it possible to achive that?

I also have yup validation. Is it possible to validate editor with yup and tell RHF that input is valid or not?

**Comment by nicolasthy** [comment]
Did you end up finding a solution to this?
